ci-operator/jobs/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-master-presubmits.yaml

```diff
@@ -14,6 +14,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-kubevirt-hyperconverged-cluster-operator-master-ci-index
     rerun_command: /test ci-index
+    run_if_changed: ^(build/*|cmd/*|deploy/crds/*|deploy/index-image/*|hack/*|pkg/*|Makefile|go\.mod|go\.sum)$
```

Comment:
1. Looking at other `run_if_changed` examples in `openshift/release`, it appears this needs to be an actual regex and not a glob, so `.*` instead of `*`.
2. I'm afraid that explicitly enumerating the triggering paths is too fragile, e.g. it can easily be overlooked when adding or renaming a folder in HCO. A much safer approach would have been to enumerate the paths for which a job should be skipped. Unfortunately such an option doesn't seem to exist, but we need to consider whether to request such a feature and wait for it to be implemented, vs. the risk/benefit of using it as is.

Reply:
:+1: fotr 1

2 - I agree but this is what we have...